packages/block-library/src/form-input/block.json

```diff
@@ -67,6 +67,9 @@
 			"__experimentalDefaultControls": {
 				"radius": true
 			}
+		},
+		"interactivity": {
+			"clientNavigation": true
```

Comment:
I wouldn't mark these new form blocks are compatible until we know more about how they work.